_posts/2021-09-29-signed-immutables-bug.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+published: true
+title: 'Signed Immutables Bug'
+date: '2021-09-29'
+author: Solidity Team
+category: Security Alerts
+---
+
+On September 28th, 2021, the Solidity team discovered that
+for immutable variables of a signed integer type shorter than 256 bits,
+sign extension (cleanup) of its value is not always properly performed.
+
+Inline assembly is required to access the value in its unclean state.
```

Comment:
```suggestion
To access the value in its unclean state, inline assembly is required.
```
The beginning sounded too technical

Reply:
Or what about something like: To our knowledge, the value can only be accessed in its unclean state when using inline assembly.